src/client.py

```diff
@@ -4,44 +4,42 @@
 import sys
 
 
```

Comment:
Nice work, looks clean.  Don't forget to use a couple blank lines once in a while to make it easier to read...like before lines 12, 29, and 34